scripts/release_support/image-based-commit-tag.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# export CORTX_IMAGE and GIT_TAG variables with appropriate values
+# CORTX_IMAGE - Any CORTX service image as input. Reference: https://github.com/Seagate/cortx-re/pkgs/container/cortx-rgw
+# GIT_TAG - Tag to be tagged to commits
+
+function get_commit_hash() {
+    component="$1"
+    for commit_detail in ${component_commit_details//,/ }
+    do
+        component_name=$(awk -F":" '{print $1}' <<< "$commit_detail")
+        if [ "$component_name" == "$component" ]; then
+            commit=$(awk -F":" '{print $2}' <<< "$commit_detail")
+            echo "$commit"
+        fi
+    done
+}
+
+if [ -z "$CORTX_IMAGE" ]; then 
```

Comment:
We should add support for all three images so that all CORTX components are tagged in a single execution.

Reply:
All CORTX images has commit hashes for all components so any image provided is good for tagging.